gatsby-browser.js

```diff
@@ -1,2 +1,3 @@
-import "./src/css/style.css";
-import "./src/css/global.css";
+import "./src/global.css";
+
+console.log("loaded gatsby-browser.js");
```

Comment:
Is this still needed?

Reply:
The import is needed to remove the border but I can remove the console.log